src/ducks/client/mobile.js

```diff
@@ -48,12 +48,12 @@ export const getManifestOptions = manifest => {
   }
 }
 
-const isRevoked = async client => {
+export const isRevoked = async client => {
   try {
     await client.stackClient.fetchInformation()
     return false
   } catch (err) {
-    if (err.message && err.message.indexOf('Client not found')) {
+    if (err.message && err.message.indexOf('Client not found') > -1) {
```

Comment:
By the way, when the code is `if (condition) { return true } else { return false }`, I prefer `return conditon` (or `return !!condition` for forcing a boolean).

Reply:
```
return includes(err.message, 'Client not found')
```